src/features/unified-share-modal/UnifiedShareModal.js

```diff
@@ -709,6 +716,8 @@ class UnifiedShareModal extends React.Component<Props, State> {
                     {...extendedModalProps}
                 >
                     <LoadingIndicatorWrapper isLoading={isFetching} hideContent>
+                        {showShareRestrictionWarning && allShareRestrictionWarning}
```

Comment:
Will this be consider as a breaking change?

Reply:
Good question. I'd recommend checking with the teams that consume USM.

Reply:
hmm actually this won't be since the text is optional.